### DISCUSSION TITLE
How to set up Colima to work with `testcontainers-node`?

### DISCUSSION THREAD

**Discussion by wujekbogdan** [testcontainers/testcontainers-node#688] [Q&A]
I managed to get it working by setting the following variables:

``` bash
export TESTCONTAINERS_DOCKER_SOCKET_OVERRIDE=/var/run/docker.sock
export TESTCONTAINERS_HOST_OVERRIDE=$(colima ls -j | jq -r '.address')
export DOCKER_HOST="unix://${HOME}/.colima/default/docker.sock"
```

together with the following config:

``` yml
network:
  address: true
```

Only then my jest tests are running.

The problem is the following line:

``` bash
export TESTCONTAINERS_HOST_OVERRIDE=$(colima ls -j | jq -r '.address')
```

Colima daemon boots at system startup thanks to a brew service:

``` bash
brew services stop colima
```

So setting `TESTCONTAINERS_HOST_OVERRIDE` it in `~/.zshrc` doesn't guarantee that Colima has already started when `~/.zshrc` is being processed.

Is my setup overcomplicated? Is there an easier way to get Colima to work with `testcontainers-node`?

**Comment by cristianrgreco** [comment]
Hi @wujekbogdan, running testcontainers-node with Colima is documented here: 
https://node.testcontainers.org/supported-container-runtimes/#colima
